src/java/com/puppetlabs/ssl_utils/ExtensionsUtils.java

```diff
@@ -616,8 +619,10 @@ private static Object asn1ObjToObj(ASN1Encodable asn1Prim)
             X500Name name = (X500Name) asn1Prim;
             return name.toString();
         } else {
-            // Return the raw data if there's no clear method of decoding
-            return asn1Prim.toASN1Primitive().getEncoded();
```

Comment:
I know you probably did this based on a conversation we were having about it but I'm having second thoughts about this now, sorry.  This part wasn't specifically needed to address the problem that was reported for SERVER-816, right?  If not, I'm thinking maybe it would be better to just let this return the raw encoded value and let the consumer convert to Base64 for output if it wants.  Just thinking that maybe down the road there would be some consumer of this data that might actually want to use it in its raw form without having to Base64 decode it.  @nwolfe - what do you think?